static/styles/help/help.scss

```diff
@@ -198,7 +198,7 @@ fieldset.switch_wrapper {
 // contact cards
 .contact-card {
   .icon-card__content {
-    background-color: $accentColor;
+    background-color: $primaryColor;
```

Comment:
isn't it possible to set accentColor to primaryColor for the brandenburg theme only?

Reply:
We really wanted to set this color for all themes, including main SC, because it looks better than it does right now.